content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -573,9 +573,22 @@ We have a lot of features in our roadmap. An abbreviated high level list is incl
 
 ### CRI-ContainerD
 
-{{< glossary_tooltip term_id="containerd" >}} is another OCI-compliant runtime that recently graduated as a {{< glossary_tooltip text="CNCF" term_id="cncf" >}} project. It's currently tested on Linux, but 1.3 will bring support for Windows and Hyper-V. [[reference](https://blog.docker.com/2019/02/containerd-graduates-within-the-cncf/)]
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
 
-The CRI-ContainerD interface will be able to manage sandboxes based on Hyper-V. This provides a foundation where RuntimeClass could be implemented for new use cases including:
+{{< glossary_tooltip term_id="containerd" >}} is another OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 is the first version tested with ContainerD on Windows. It's still a work-in-progress, and you can track progress in [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001). 
+
+ContainerD on Windows is suitable for development purposes only for a few reasons:
+
+- ContainerD development for Windows is still in progress. Tests rely on builds from the master branch. ContainerD 1.4 is expected to work at feature parity with Docker, but it will ship sometime after Kubernetes v1.18.
+- Group-Managed Service Accounts are not yet implemented - see [containerd/cri#1276](https://github.com/containerd/cri/issues/1276).
```

Comment:
[Avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) (these are fine in blog posts though).